app-api-common/recipe/src/main/java/org/codeforvictory/android/recipes/data/api/recipe/RecipeApiSource.kt

```diff
@@ -0,0 +1,17 @@
+package org.codeforvictory.android.recipes.data.api.recipe
+
+import org.codeforvictory.recipes.domain.common.model.meal.Meal
+
+class RecipeApiSource(
+    private val adaptToMeal: AdaptToMeal = AdaptToMeal(),
+    private val recipeService: RecipeService = RecipeServiceProvider().value,
```

Comment:
talking offline Cesar and I:

* Not using default arguments
* We do not need RecipeServiceProvider
* We can keep RetrofitProvider to have a single place where we create the retrofit config

Reply:
Addressed: https://github.com/CodeForVictory/recipes-mvi-playground/pull/13/commits/4606ac9ce1e6d7155cba6ed2f1f3145ed6ee3929